pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -86,6 +86,15 @@ type PrometheusSpec struct {
 
 	// +optional
 	Ingress *IngressSpec `json:"ingress,omitempty"`
+
+	Container *corev1.Container `json:"container,omitempty"`
+	// +optional
+	Config *Configuration `json:"config,omitempty"`
+}
+
+// Config  is the the desired state of Prometheus Configuration
+type Configuration struct {
+	ConfigMapRef string `json:"externalConfigMap,omitempty"`
```

Comment:
Please add the note to explain What would this attribute do for TidbMonitor and What requirements this configmap should meep.

Reply:
If we only reference confi map name in Tidbmonitor Spec, we should point out that the namespace the external configmap placed in should be as same as the its Tidbmonitor.